components/exercises/chapter-4-oscar/SumOfARange.tsx

```diff
@@ -0,0 +1,211 @@
+                >
+                  <option value='1'>1</option>
+                  <option value='2'>2</option>
+                  <option value='3'>3</option>
+                  <option value='4'>4</option>
+                  <option value='5'>5</option>
+                  <option value='6'>6</option>
+                  <option value='7'>7</option>
+                  <option value='8'>8</option>
+                  <option value='9'>9</option>
+                  <option value='10'>10</option>
+                </select>
+                <div className='pointer-events-none absolute inset-y-0 right-0 flex items-center px-2 text-gray-700'>
+                  <svg
+                    className='fill-current h-4 w-4'
+                    xmlns='http://www.w3.org/2000/svg'
+                    viewBox='0 0 20 20'
+                  >
+                    <path d='M6.293 9.293a1 1 0 011.414 0L10 11.586l2.293-2.293a1 1 0 111.414 1.414l-3 3a1 1 0 01-1.414 0l-3-3a1 1 0 010-1.414z' />
+                  </svg>
+                </div>
+              </div>
+              <p>to</p>
+              <div className='w-40 relative'>
+                <select
+                  value={showAnimals.end}
+                  onChange={handleChangeEnd}
+                  className='block appearance-none w-full bg-white border border-gray-300 hover:border-gray-400 px-4 py-2 pr-8 rounded shadow leading-tight focus:outline-none focus:ring focus:border-blue-500'
+                >
+                  <option value='1'>1</option>
+                  <option value='2'>2</option>
+                  <option value='3'>3</option>
+                  <option value='4'>4</option>
+                  <option value='5'>5</option>
+                  <option value='6'>6</option>
+                  <option value='7'>7</option>
+                  <option value='8'>8</option>
+                  <option value='9'>9</option>
+                  <option value='10'>10</option>
+                </select>
+                <div className='pointer-events-none absolute inset-y-0 right-0 flex items-center px-2 text-gray-700'>
+                  <svg
+                    className='fill-current h-4 w-4'
+                    xmlns='http://www.w3.org/2000/svg'
+                    viewBox='0 0 20 20'
+                  >
+                    <path d='M6.293 9.293a1 1 0 011.414 0L10 11.586l2.293-2.293a1 1 0 111.414 1.414l-3 3a1 1 0 01-1.414 0l-3-3a1 1 0 010-1.414z' />
+                  </svg>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div className='flex flex-col flex-[0.5] ml-2'>
+          <p className='mt-3 mb-2'>Sum of animals</p>
+
+          <p className='flex flex-row justify-start py-1'>
+            {arraySum(calcRangeSum(showAnimals.start, showAnimals.end))}
+          </p>
+        </div>
+      </div>
+    </div>
+  );
+}
```

Comment:
I've been playing with the exercise on the live preview and it is a bit hard to understand what is going on. I think it would be clearer by:
1. Setting an exercise description, explaining what is going on
2. Adding the value of each animal next to their name, so the sum makes sense to the viewer